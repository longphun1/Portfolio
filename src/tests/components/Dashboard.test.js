import React from 'react';
import { shallow } from 'enzyme';
import  expenseDashboard  from '../../components/dashboard';

test('should render dashboard correctly', () => {
    const wrapper = shallow(<expenseDashboard/>)
    expect(wrapper).toMatchSnapshot();
})