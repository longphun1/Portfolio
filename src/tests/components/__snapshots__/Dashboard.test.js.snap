// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard correctly 1`] = `
<div>
  <h1>
    Dashboard
  </h1>
</div>
`;
